test/modules/aggregation-layers/grid-aggregation-layer.spec.js

```diff
@@ -214,7 +215,7 @@ test('GridAggregationLayer#CPUvsGPUAggregation', t => {
         data: fixture.data,
         getWeight: x => x.weight1[0],
         aggregation: 'SUM',
-        cellSize: 5000,
+        cellSize: 50,
```

Comment:
Wasn't this set to `5000` to reduce test runtime?

Reply:
Needed one test case with smaller cell size,  otherwise, everything gets aggregated to the same cell.
Earlier running time was an issue on CI only (browser tests were fast), but now I re-tested, test finished quickly.